lib/capybara/screenshot/diff.rb

```diff
@@ -55,12 +55,7 @@ module Diff
       mattr_accessor(:tolerance) { 0.001 }
 
       AVAILABLE_DRIVERS = Utils.detect_available_drivers.freeze
-      begin
-        require "minitest"
-        ASSERTION = Minitest::Assertion
-      rescue
-        ASSERTION = RuntimeError
-      end
+      ASSERTION = Utils.detect_test_framework_assert
```

Comment:
You might consider making Capybara::Screenshot::Framework::MiniTest and Capybara::Screenshot::Framework::Default classes and having a single place where you do the framework check to instantiate the right class and then delegate to that instance for all of these sorts of things. 

Thanks for the quick response to the Issue!